src/gui/gui.cc

```diff
@@ -1465,6 +1458,7 @@ this setting may not have any effect for you.)"));
         ShowHelpMarker(_(R"(Emulates an installed 8MB system,
 instead of the normal 2MB. Useful for working
 with development binaries and games.)"));
+        changed |= ImGui::Checkbox(_("OpenGL GPU"), &settings.get<Emulator::SettingHardwareRenderer>().value);
```

Comment:
This will have to be a "toggled on reboot" state for now, at least until sstates are implemented. Software and hardware renderers store state in completely different ways unlike eg the CPU cores. The Display struct and future structures will aim to make the problem less bad.

Reply:
Then maybe we need the same warning as for interpreter / dynarec popping up?